src/containers/forms/PlanForm/index.tsx

```diff
@@ -609,6 +609,7 @@ const PlanForm = (props: PlanFormProps) => {
             <hr className="mb-2" />
             <Button
               type="submit"
+              id="submit-button"
```

Comment:
Added an id here to be able to more easily select this element

Reply:
Are there any other submit buttons on the page? Could there be? If yes, let's make this id string more contextually specific, if no then this is good.

Reply:
I'll change this to `planform-submit-button`